frontend/views/components/Menu/MenuItem.vue

```diff
@@ -16,7 +16,7 @@ export default {
   inheritAttrs: false,
   methods: {
     handleSelect () {
-      this.Menu.handleSelect(this.$attrs.itemId) // Use $attrs to access passed props
+      this.Menu.handleSelect(this.$attrs['item-id']) // Use $attrs to access passed props
```

Comment:
Something weird here.
Normally we don't inject an object with functions from the parent to be executed in the child.
We get the click from child then pass to the parent the event to execute the function.
Child:
list-item(@click="$emit('handle-select', $attrs['item-id])")
Parent:
MenuItem(:handle-select=”something”)